datafusion/common/src/parsing.rs

```diff
@@ -0,0 +1,17 @@
+pub fn is_system_variables(variable_names: &[String]) -> bool {
```

Comment:
FYI the reason the "RAT" test is failing on this PR is that all files need to have the Apache license at the start. 

What would you think about putting this code into the existing module https://github.com/apache/arrow-datafusion/blob/master/datafusion/physical-expr/src/var_provider.rs rather than a new module?